pipeline/pandaxiii_MC/readout-140kg.rml

```diff
@@ -1,137 +1,51 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
-
-<readouts>
-
-    <globals>
-        <variable name="PITCH" value="3" overwrite="false"/>
-        <!-- The pitch value in mm -->
-        <variable name="MODULES" value="10" overwrite="false"/>
-        <!-- The number of modules per axis MxM -->
-        <variable name="CHANNELS" value="64" overwrite="false"/>
-        <!-- The number of pixels per side PxP -->
-    </globals>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<readouts><globals><variable name="PITCH" value="3" overwrite="false"/><!-- The pitch value in mm --><variable name="MODULES" value="10" overwrite="false"/><!-- The number of modules per axis MxM --><variable name="CHANNELS" value="64" overwrite="false"/><!-- The number of pixels per side PxP --></globals>
```

Comment:
The pre-commit bot is configured to automatically format xml files yes, I think it makes sense to do so because in my experience people misformat them very often. It can be a bit annoying but I think it's worth it. I would not disable the automatic formatting of xml files.

If you want, you can add a custon exception to the rule in the following file: https://github.com/rest-for-physics/framework/blob/master/.pre-commit-config.yaml.

Reply:
It seems that this  formatting have been generated by the previous version of pre-commit hooks, most likely because someone still have it in their local branch, note that these changes cannot bee easily removed, you have to do it manually or checking out a new branch.

On the other hand, I have no concerns from what current pre-commit CI is doing.